angular-route/templates/_.client.route.js

```diff
@@ -0,0 +1,5 @@
+$stateProvider.
+  state('<%= slugifiedName %>', {
+    url: '/<%= slugifiedRoutePath %>',
+    templateUrl: 'modules/<%= slugifiedModuleName %>/client/views/<%= slugifiedViewName %>.client.view.html'
+  }).
```

Comment:
Where do this get pulled into that makes it a partial? Also, why does it end with a .?